src/Header/Account/Account.jsx

```diff
@@ -108,6 +121,8 @@ export default class Account extends Component {
 				.then(
 					function(data) {
 						if (this.props.onLoginStatusChecked) {
+							data.links = { ...consultationsResponsesLink, ...data.links };
```

Comment:
Here, we're always prepending the consultationsResponsesLink, without checking whether you're logged in or not. Whereas with idam (see line 106) we're only showing the link when you're logged in. Is that expected?

Reply:
It always gets prepended, but shouldn't show due to the logged in condition in the render further down on ln149.